js/src/views/Application/Extension/store.js

```diff
@@ -60,19 +60,27 @@ export default class Store {
   }
 
   @action testInstall = () => {
-    this.shouldInstall = this.readStatus();
+    this.readStatus()
```

Comment:
MobX sometimes does things a bit funny when dealing with promises, i.e. not firing updates when inside a promise chain. Generally it would be preferable to...

```
@action setInstallStatus = (shouldInstall) => {
  this.shouldInstall = shouldInstall;
}

testInstall = () => {
  this.readStatus().then(this.setInstallStatus);
}
```